Automattic/wp-calypso
# Update/status cell design

This PR adopt the status column design to match the new design.
![image](https://github.com/Automattic/wp-calypso/assets/47489215/504e669f-e025-4842-8b5e-81660de33803)


## Proposed Changes

* Redesign status cell
* Count the number of alerts requiring attention and display in the header

## Testing Instructions



* go to `/domains/manage?flags=domains/management`
* Verify the design matches p9Jlb4-8DU-p2 
* Verify the correct number of alerts is displayed in the header, it should match what's in prod

## Pre-merge Checklist
